### PR TITLE
social sharing quick guide

### PR DESCRIPTION
**Pull request by mchacong**
Added quick guide to enable social sharing per https://track.akamai.com/jira/browse/AMPWEB-2647

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -59,11 +59,11 @@
 
 .UI
 * xref:amp-react:index.adoc[React]
-** xref:amp-react:1-overview.adoc[Overview]
```
Comment (JeanCarlosChavarriaHughes) [inline modules/ROOT/nav.adoc:62]: @mchacong shouldn't we let the Overview page active in the index ?

File: modules/amp-react/pages/6-share.adoc [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+.amp-share {
+    display: none;
+}
+----
+
+== How to use built-in social sharing UI
+
+Player offers a built-in share panel and buttons along with callbacks which can be connected to customer's favorite social provider.
+
+Player offers `share` event callback to hook users choice, within in payload can be use to perform an action such like prompting a redirect to the social network url or show a custom overlay.
```
Comment (JeanCarlosChavarriaHughes) [inline modules/amp-react/pages/6-share.adoc]: @mchacong 
`within in the payload` sounds duplicated, maybe `within the payload` is better.

File: modules/amp-react/pages/6-share.adoc [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+- Snapchat
+- Embed
+- Email
+- Other
+
+NOTE: Ask your Akamai representative if need to add a new option. 
+
+== Configuring social providers list
+
+If required, the list of providers listed in the share panel UI element can be modified whether to add certain options or remove unneeded options on `providers` array.
```
Comment (JeanCarlosChavarriaHughes) [inline modules/amp-react/pages/6-share.adoc]: @mchacong this seems a typo.
`either` instead of `whether` 

??

**Review by JeanCarlosChavarriaHughes — COMMENTED** [review]
Please check the comments.

File: modules/ROOT/nav.adoc (hunk 1 above)
Comment (mchacong) [reply modules/ROOT/nav.adoc:62]: no longer need this

File: modules/amp-react/pages/6-share.adoc (hunk 2 above)
Comment (mchacong) [reply modules/amp-react/pages/6-share.adoc]: fixed

File: modules/amp-react/pages/6-share.adoc (hunk 3 above)
Comment (mchacong) [reply modules/amp-react/pages/6-share.adoc]: fixed typo

**Review by JeanCarlosChavarriaHughes — APPROVED** [review]
This looks good. We need to wait the release of amp-web-sdk 9.1.27

**Pull request merged**